t/nvmept_fdp.py

```diff
@@ -674,6 +725,72 @@ def check_all_ruhs(dut):
         "test_class": FDPTest,
         "success": SUCCESS_NONZERO,
     },
+    # Specify invalid options related to dataplacement scheme
+    ## using old and new sets of options
+    {
+        "test_id": 302,
+        "fio_opts": {
+            "rw": 'write',
+            "bs": 4096,
+            "io_size": 4096,
+            "verify": "crc32c",
+            "fdp": 1,
+            "fdp_pli": 3,
+            "fdp_pli_select": "scheme",
+            "output-format": "normal",
+        },
+        "test_class": FDPTest,
+        "success": SUCCESS_NONZERO,
+    },
+    {
+        "test_id": 303,
+        "fio_opts": {
+            "rw": 'write',
+            "bs": 4096,
+            "io_size": 4096,
+            "verify": "crc32c",
+            "fdp": 1,
+            "fdp_pli": 3,
+            "plid_select": "scheme",
+            "output-format": "normal",
+        },
+        "test_class": FDPTest,
+        "success": SUCCESS_NONZERO,
+    },
+    # write to multiple PLIDs using scheme selection of PLIDs
+    ## using old and new sets of options
+    {
+        "test_id": 400,
+        "fio_opts": {
+            "rw": 'randwrite',
+            "bs": 4096,
+            "size": 1048576,
+            "io_size": 1073741824,
```

Comment:
Consider making this just a very simple test with a single write to each of the PLIDs.

Something along these lines:
```
"io_size": 3*4096
"rw": `write:524288`
```

Fio will issue sequential write operations with a 524288-byte hole between each write. Then you should end up with a single write to each PLID.

Reply:
Simplifying the test would be great.

FYI, there was an issue which gets me to make tests a bit complex. I figured out that the test assumes that `ruamw` decreases whenever a write occurs on plids. In my case, I ran the test against a real device supporting FDP. But the device did not work in that way. (※ `ruamw` decreases after multiple writes occur and the `FIO_FDP_MAX_RUAMW` was large enough, 1,571,328)

In your test environment, `ruamw` of `1`, `2`, and `3` may be updated (`io_size/bs`=262,144, multiple of 32) times in total. The result value of `ruamw`, 32, can be accepted as a wrong value even though the value is correct. Simplifying the test can also resolve [the failure of the test](https://github.com/axboe/fio/pull/1757#issuecomment-2108070323).